### ISSUE TITLE
Did you serve a mission? 

### ISSUE THREAD

**Issue by hamstu** [kentcdodds/ama#31]
If so, where?

PS. Hi from a fellow Mormon! It's always nice to get to know more people in our industry in the church. Let's be friends! :+1:

**Comment by kentcdodds** [comment]
Yes! I absolutely did serve a mission. I served my mission in the [Missouri Independence Mission](http://missionhome.com/?missions=missouri-independence-mission) it was totally awesome. Here are some (maybe) interesting stats off the top of my head:

Areas served:

1. Topeka, Kansas
2. Cameron/Chillicothe, Missouri
3. Salina, Kansas
4. North Kansas City, Missouri
5. Topeka, Kansas (again)
6. Leawood, Kansas

I had 16 companions. I served as a Trainer, District Leader, and Zone Leader. I served from February 2008 to February 2010. It was awesome. I loved my mission. I'm waiting for a JavaScript conference to happen in Kansas City so I can try to get accepted to speak and have an excuse to go back :-)

Where did you serve?

**Comment by hamstu** [comment]
Thanks for the response, Kent!

I served in the wonderfully temperate [California Sacramento Mission](http://missionhome.com/?missions=california-sacramento-mission) from February 2009 to February 2011.

I kept a blog while serving, which I was going to link to, but upon checking the URL I got an `Error establishing database connection` — whoops! I don't know how long that's been there, so I'll have to look into that, haha.

I served mostly in and around the city of Sacramento. I had a number of companions (10 or 12? I'll have to dust off a journal), most from the good old USA. I was blessed to teach some amazing people. I also had a chance to serve in the same leadership capacities you mentioned, plus a really fun stint as an *Office Elder*, where I got to design a new welcome packet for incoming missionaries and build an app to get missionary contact details from *iMOS* onto the mission cell phones. (They still use it today, so it's like a part of me's still there `</nerd>`)

Haha, I love your conference trip plan. I've only been back to my mission once, before I got married. We're a little more settled now with a 7 month old boy, but I'm hoping to take the family back again one day.
